app/server/logging/jsondb/src/main/java/io/syndesis/server/logging/jsondb/controller/PodLogMonitor.java

```diff
@@ -66,13 +66,13 @@ public PodLogMonitor(ActivityTrackingController logsController, Pod pod) throws
             throw new IOException("Could not determine the pod name");
         }
 
-        Map<String, String> annotations = pod.getMetadata().getAnnotations();
-        this.integrationId = annotations.get(OpenShiftService.INTEGRATION_ID_ANNOTATION);
+        Map<String, String> labels = pod.getMetadata().getLabels();
+        this.integrationId = labels.get(OpenShiftService.INTEGRATION_ID_ANNOTATION);
```

Comment:
`INTEGRATION_ID_ANNOTATION`, shouldn't this be `INTEGRATION_ID_LABEL` ? (and removing the constant about annotations ?)